src/components/images/gallery/GalleryItem.js

```diff
@@ -1,11 +1,9 @@
 // @flow
 import React from 'react';
-import shortid from 'shortid';
-import { IoIosMore } from 'react-icons/io';
 import { Link } from 'react-router-dom';
-import Tag from '../utility/Tag';
-import type { Image } from '../../models/Image';
+import type { Image } from '../../../models/Image';
 import './Gallery.scss';
```

Comment:
Not that important but I'd like to have the scss imports at the bottom of the import list.